### ISSUE TITLE
Add first exercise sub-step to course spec

### ISSUE THREAD

**Issue by richierocks** [datacamp/authoring#102]
For courses in the middle of a track, some instructors find it difficult to know where to begin there course.

I propose updating step three of the course spec so that it has 3 parts.

3.1: Write the first exercise, so you know where the course begins
3.2: Write the last exercise, so you know where the course ends (as before)
3.3: Write some exercises that detail how you get from the start to the end (as before)

Even for courses which aren't in the middle of the track, I think that it can be useful to know the starting point. (It should help clarify prerequisites, and if instructors are throwing in material that is too tricky, we'll discover the problem early.)

In the directory `courses/design`, this requires

1. An update to `template.md` to describe this sub-step.
1. A new mini-manual, `exercises-first.md` with examples, etc.
1. Minor updates to `exercises-capstone.md` and `exercises-examples.md` to renumber them.

**Comment by ismayc** [comment]
I'm in favor of this but I think pretty soon we need to come to a consensus on course specs or begin to version them. We really need to get feedback from each CD on what exactly they are looking for with the specs to come to a concensus.

It's becoming more and more confusing to try to figure out what instructors were asked to do while also matching up with what CDs are looking for in terms of specs. Makes me wonder if maybe a README file isn't the best long-term solution here... It would be nicer if as we updated the course specs template, if instructors using older versions could also see the requested changes.

**Comment by gvwilson** [comment]
I agree with both of you - can we put this on the agenda for the work week?
